src/util/CMakeLists.txt

```diff
@@ -77,3 +146,4 @@ ide_split_sources(util)
 
 target_include_directories(util PRIVATE ${UTIL_INCLUDES} ${LIBGIT2_DEPENDENCY_INCLUDES} PUBLIC ${libgit2_SOURCE_DIR}/include)
 target_include_directories(util SYSTEM PRIVATE ${LIBGIT2_SYSTEM_INCLUDES})
+target_link_libraries(util PUBLIC libgit2_https_backend libgit2_regex libgit2_zlib $<$<BOOL:${USE_NTLMCLIENT}>:ntlmclient> $<$<BOOL:${USE_THREADS}>:Threads::Threads> libgit2_link_libraries)
```

Comment:
I still don't understand this yet. Is there a library called `Threads` that I don't see declared? 🤔